ReconstructionByHeight.java

```diff
@@ -0,0 +1,24 @@
+class Solution {
+
+    // Time Complexity : 0(nlogn) + 0(n^2) where  is the no. of elements in the array. nlogn is for sorting and n^2 is for rearranging the descending array in result list as per their indices
+// Space Complexity : 0(1) we don't conside space for hashmap because in any case it will hold only 26 alphabets
```

Comment:
But result list is used here and our return type Is int[][] so space will be O(N)